wftests/warrior_tests/suites/selenium_func_tests/gen_test_suite.xml

```diff
@@ -117,13 +117,13 @@
 			<onError action="next"/>
 			<impact>impact</impact>
 		</Testcase>
```

Comment:
Why are these steps being commented out? If the data is outdated, have new tickets been created to address the issue?

Reply:
Hi @sanikakulkarni 
We have already created the Jira ticket(WAR-2197) for commented code and updated the same in the respective ticket.